proselint/checks/lexical_illusions/misc.py

```diff
@@ -21,11 +21,6 @@ def check(text):
     """Check the text."""
     err = "lexical_illusions.misc"
     msg = u"There's a lexical illusion here: a word is repeated."
+    regex = r"\b(\w+)\b\s\1"
```

Comment:
There are a few instances where repeated words are perfectly acceptable. For example, one day I was commenting on student writing and I noticed that the student used the word "that" instead of "which". It's a hard distinction for many, and I don't blame them. So here I am now, telling you that that "that" that that student wrote ought to have been a "which".

Reply:
See also https://en.wikipedia.org/wiki/Buffalo_buffalo_Buffalo_buffalo_buffalo_buffalo_Buffalo_buffalo

Reply:
https://en.wikipedia.org/wiki/James_while_John_had_had_had_had_had_had_had_had_had_had_had_a_better_effect_on_the_teacher